### PR TITLE
feat: Crunchbase - Get Data From Entity

### PR DESCRIPTION
**Pull request by FlorentLvr**
This PR resolves https://github.com/jupyter-naas/awesome-notebooks/issues/1527

### CODE REVIEW

**Comment by FlorentLvr** [comment]
Hey @aybruhm,

I wanted to give you a heads up on an issue I created, but unfortunately I haven't had the time to work on it. Would you be interested in giving it a go?
Basically, we need to extract some data from a company, which is going to be really useful for scoring them in future marketing campaigns. To do this, you'll need to create an account on Crunchbase. But don't worry, they offer a free plan for users.
Let me know if you're up for it or have any questions!

Thanks,

File: Crunchbase/Crunchbase_Get_Data_From_Entity.ipynb [hunk 1]
```diff
@@ -0,0 +1,241 @@
...
+  },
+  {
+   "cell_type": "markdown",
+   "id": "577a9dd1-9ecf-4ab2-80dd-d63d71b094e3",
+   "metadata": {
+    "papermill": {},
+    "tags": []
+   },
+   "source": [
+    "**References:**\n- [Crunchbase Entity Lookup API](https://data.crunchbase.com/docs/examples-entity-lookup-api)\n- [Crunchbase API Documentation](https://data.crunchbase.com/docs/using-the-api)"
```
Comment (srini047) [inline Crunchbase/Crunchbase_Get_Data_From_Entity.ipynb:66]: @FlorentLvr Getting a 404 page when clicking the link.

**Review by srini047 — COMMENTED** [review]
@FlorentLvr The template needs some changes.

**Comment by FlorentLvr** [comment]
@srini047, I am not sure @aybruhm worked on it. Can you take over?

**Comment by srini047** [comment]
> @srini047, I am not sure @aybruhm worked on it. Can you take over?

I'm not sure because I haven't tried Cruchbase but I would like to try and revert to you soon regarding the progress.

**Comment by FlorentLvr** [comment]
@Dr0p42, could you please provide some guidance on this issue🙏 ?

**Comment by FlorentLvr** [comment]
As discussed with @Dr0p42, we won't develop templates for now as this is a paid API.

**Pull request closed without merging**